lib/matplotlib/axes/_base.py

```diff
@@ -4096,19 +4096,45 @@ def pick(self, *args):
         martist.Artist.pick(self, args[0])
 
     def get_default_bbox_extra_artists(self):
+        """
+        Return a default list of artists that are used for the bounding box
+        calculation.  Artists are excluded either by not being visible
+        or ``artist.inbbox = False``.
+        """
         return [artist for artist in self.get_children()
-                if artist.get_visible()]
+                if (artist.get_visible() and artist.inbbox)]
 
-    def get_tightbbox(self, renderer, call_axes_locator=True):
+    def get_tightbbox(self, renderer, call_axes_locator=True,
+            bbox_extra_artists=None):
```

Comment:
please do not add this extra kwarg.

Reply:
Sorry, I screwed up the PR during a rebase...  The reason for this is so that the artist logic that was in `backend_bases.print_figure` could be refactored out and reused.   So this needs to be here to make that 
function work.  Could make a private kwarg (is there such a thing?)